charmcraft/templates/init-simple/metadata.yaml.j2

```diff
@@ -21,3 +21,5 @@ resources:
   httpbin-image:
     type: oci-image
     description: OCI image for httpbin (kennethreitz/httpbin)
+    # Included for simplicity in integration tests
```

Comment:
This file needs overhauling slightly - please check the template here: https://github.com/canonical/template-operator/blob/charmcraft-templates/default/metadata.yaml